### DISCUSSION TITLE
Conflicting notation of VerifiableCredential.issuer

### DISCUSSION THREAD

**Discussion by martijndebruijn** [decentralized-identity/veramo#756] [Q&A] [answered]
In the https://www.w3.org/TR/vc-data-model/#issuer spec two formats for specifying issuers are specified:

`"issuer": "https://example.edu/issuers/565049",`

or 
```
"issuer": {
    "id": "did:example:76e12ec712ebc6f1c221ebfeb1f",
    "name": "Example University"
  },
```
I have to handle the first issuer notation. However the Veramo VerifiableCredential interface requires an id attribute inside the issuer object.
How should I handle this situation?

**Comment by mirceanis** [comment, marked as answer]
The `issuer.id` property is going to be interpreted as `iss` for the credential JWT, so it's not a problem.
The current veramo release only supports JWT credentials. Json-ld support is not yet implemented, but it's coming soon, and this parameter requirement will be relaxed.

**Reply by martijndebruijn** [reply]
Do you know when the version with Json-Ld support will be released?

**Reply by mirceanis** [reply]
I can't provide a release date, but this feature is my main focus now.

Initially it will support only 2 signature suites suitable for Ed25519 and secp256k1 keys that are already well supported by veramo.

**Reply by mirceanis** [reply]
You can follow progress here: https://github.com/uport-project/veramo/pull/525

**Comment by martijndebruijn** [comment]
Super. Thanks. Good to know.
